marklogic-data-hub-central/ui/src/components/selected-facets/selected-facets.tsx

```diff
@@ -122,7 +122,7 @@ const SelectedFacets: React.FC<Props> = (props) => {
               data-cy='clear-date-facet'
               data-testid='clear-date-facet'
             >
-              <Icon type='close'/>
+              <Icon type='close' className={styles.clearIcon}/>
```

Comment:
To move the icon to the right, the Icon element can be moved after the text.

Reply:
@brucean52 thanks for noticing that, some how I missed it. Please review it now.